pkg/apis/cluster/v1alpha1/machine_types.go

```diff
@@ -183,3 +185,16 @@ func (MachineSchemeFns) DefaultingFunction(o interface{}) {
 	// set default field values here
 	log.Printf("Defaulting fields for Machine %s\n", obj.Name)
 }
+
+func ParseMachineYaml(file string) (*Machine, error) {
```

Comment:
This seems awfully specific to the machine actuator tests. 

It is assuming that the file contains only one machine object and no other objects. Perhaps it is better to co-locate such logic with the tests and then promote to a common location if other code actually uses it?

Reply:
This will be removed in a future revision of this PR.